packages/-ember-data/addon/setup-container.js

```diff
@@ -31,7 +30,6 @@ function initializeStore(registry) {
   registerOptionsForType.call(registry, 'adapter', { singleton: false });
   registry.register('serializer:-default', JSONSerializer);
   registry.register('serializer:-rest', RESTSerializer);
-  registry.register('adapter:-rest', RESTAdapter);
```

Comment:
Out of curiosity - this isn't a "breaking" change, right?  (still acclimating)

Reply:
Correct. The existence of this adapter was for tests and private. To use it folks would have had to discover it and looked it up via -rest manually. Given the easy of importing RESTAdapter and no evidence of usage I don’t believe we need to treat this as intimate.